start.js

```diff
@@ -0,0 +1,122 @@
+const http = require("http");
+const soldierRoute = require("./routes/soldiers.js");
+const dutiesRoute = require("./routes/duties");
+const justiceBoardRoute = require("./routes/justiceBoard");
+const configs = require('./config');
+const MongoClient = require('mongodb').MongoClient;
+const utils = require("./utility/utility");
+const soldierDb = require("./collections/soldiers");
+
+// Connection URL
+const mongoUrl = configs.dbUrl;
+
+// Database Name
+const dbName = configs.dbName;
+
+let db = 1;
```

Comment:
global variables are the devil. if you really need this to be accesaible implement a getter method. but even that is not necessary